core/trino-spi/src/main/java/io/trino/spi/security/SystemAccessControl.java

```diff
@@ -253,11 +253,22 @@ default Set<String> filterCatalogs(SystemSecurityContext context, Set<String> ca
         return emptySet();
     }
 
+    /**
+     * Check if identity is allowed to create the specified schema with properties in a catalog.
+     *
+     * @throws AccessDeniedException if not allowed
+     */
+    default void checkCanCreateSchema(SystemSecurityContext context, CatalogSchemaName schema, Map<String, Object> properties)
+    {
+        denyCreateSchema(schema.toString());
```

Comment:
Why not delegate to `checkCanCreateSchema(context, schema)`?
that would make the change backward-compatible.

Reply:
proposing https://github.com/trinodb/trino/pull/15618 to address this

Reply:
You are right. That was a mistake.